Apps/Playground/iOS/LibNativeBridge.mm

```diff
@@ -7,10 +7,23 @@
 #import <Babylon/ScriptLoader.h>
 #import <Babylon/XMLHttpRequest.h>
 #import <Shared/InputManager.h>
+#import "Babylon/XMLHttpRequestApple.h"
 
 std::unique_ptr<Babylon::AppRuntime> runtime{};
 std::unique_ptr<InputManager::InputBuffer> inputBuffer{};
 
+void XMLHTTPRequestPlugin(Napi::Env env)
```

Comment:
The function name is not consistent with other plugins, though it's not completely consistent within the other plugins either. Is it possible to put this in a namespace so that you can do `Babylon::InitializeXMLHttpRequest`?